src/test/java/uk/gov/hmcts/reform/ccd/document/am/service/impl/CaseDataStoreServiceImplTest.java

```diff
@@ -74,34 +69,9 @@ void getCaseDocumentMetadata_HappyPath() {
 
         caseDocumentMetadataResponse = sut.getCaseDocumentMetadata(CASE_ID, UUID.fromString(MATCHED_DOCUMENT_ID));
         assertNotNull(caseDocumentMetadataResponse);
-        assertEquals(CASE_ID,caseDocumentMetadataResponse.get().getCaseId());
-        assertEquals(MATCHED_DOCUMENT_ID,caseDocumentMetadataResponse.get().getDocument().getId());
+        assertEquals(MATCHED_DOCUMENT_ID,caseDocumentMetadataResponse.get().getId());
     }
 
-    @Test
```

Comment:
Test case removed. Is this no longer applicable now?

Reply:
Yes, it was no longer required.